tests/Unit/Infrastructure/ErrorLoggingFileFetcherTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Unit\Infrastructure;
+
+use FileFetcher\FileFetcher;
+use FileFetcher\FileFetchingException;
+use FileFetcher\InMemoryFileFetcher;
+use Psr\Log\LoggerInterface;
+use WMDE\Fundraising\Frontend\Infrastructure\ErrorLoggingFileFetcher;
+use PHPUnit\Framework\TestCase;
+
+class ErrorLoggingFileFetcherTest extends TestCase {
+
+	public function testGivenSucceedingFileFetcher_itsContentIsReturned() {
+		$logger = $this->createMock( LoggerInterface::class );
+		$logger->method( 'error' )->with( $this->never() );
+		$errorLoggingFileFetcher = new ErrorLoggingFileFetcher(
+			new InMemoryFileFetcher( ['song.txt' => 'I\'m a little teapot'] ),
```

Comment:
You're right, and I'll change it.

But as a side note: Does our phpcs config reflect that already? When we started this project, we used the MediaWiki standard which got [changed in Sep 2016](https://www.mediawiki.org/w/index.php?title=Manual:Coding_conventions/PHP&oldid=2227991) to the more spacey style and I think our PHP code mostly adheres to the old style.